crates/transpiler/src/target/mod.rs

```diff
@@ -305,7 +305,7 @@ impl Target {
             granularity: granularity.unwrap_or(1),
             min_length: min_length.unwrap_or(1),
             pulse_alignment: pulse_alignment.unwrap_or(1),
-            acquire_alignment: acquire_alignment.unwrap_or(0),
+            acquire_alignment: acquire_alignment.unwrap_or(1),
```

Comment:
This is also an unrelated fix right? It's fine given the timeframe but it would be better as a standalone PR.

Reply:
Yeah, for this one it was a bug I caught just now which would've surfaced after #14308 merged. There was some incorrect initialization happening in the rust-side target.